README.md

```diff
@@ -214,6 +214,53 @@ async () => {
 
 Returns **[Promise](https://developer.mozilla.org/docs/Web/JavaScript/Reference/Global_Objects/Promise)&lt;[array](https://developer.mozilla.org/docs/Web/JavaScript/Reference/Global_Objects/Array)&lt;Experience>>** Promise with an array of Experience objects.
 
+#### getPlacement
+
+Returns Placement for given parameters.
```

Comment:
Method getPlacement should be added to the Typescript definition of API in /src/index.ts